client/apps/settings/client/curations/venice_section.coffee

```diff
@@ -16,14 +17,22 @@ module.exports = VeniceSection = React.createClass
     cover_image: @props.section.cover_image or ''
     published: @props.section.published or false
     release_date: moment(@props.section.release_date).format('YYYY-MM-DD') or null
+    slug: @props.section.slug or ''
 
   componentDidMount: ->
     ReactDOM.findDOMNode(@refs.container).classList += ' active'
+    @setupSlug()
+
+  setupSlug: ->
+    if !@props.section.slug and @props.section.title
+      @setState slug: _s.slugify @props.section.title
```

Comment:
This is cool! But I'm sure people will want to customize the slugs. It probably makes more sense to make this into an input field for more flexibility. I think we could check in with @owendodd about that tho.